pkgs/development/python-modules/sphinx-tippy/default.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  flit-core,
+  beautifulsoup4,
+  jinja2,
+  requests,
+  sphinx,
+  sphinxHook,
+  furo,
+  myst-parser,
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-tippy";
+  version = "0.4.3";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "sphinx_tippy";
+    inherit version;
+    hash = "sha256-JVq+4K7YCF/bmrDMWVzXpFdIrn9GYhVuUAoXo/c61j0=";
+  };
```

Comment:
fetchFromGitHub is preferred unless there is a good reason to use Pypi

Reply:
Hmmm, that's the first time I'm hearing of it. May I ask why?